crates/rolldown/src/module_finalizers/isolating/mod.rs

```diff
@@ -11,7 +11,7 @@ pub struct IsolatingModuleFinalizerContext<'me> {
 
 pub struct IsolatingModuleFinalizer<'me, 'ast> {
   pub ctx: &'me IsolatingModuleFinalizerContext<'me>,
-  pub scope: &'me AstScopes,
+  // pub scope: &'me AstScopes,
```

Comment:
I think we should either remove or add `allow(unused)` with a comment that why we need it in the future.